lib/data_update_scripts/20201217184442_append_community_to_community_name.rb

```diff
@@ -0,0 +1,11 @@
+module DataUpdateScripts
+  class AppendCommunityToCommunityName
+    def run
+      return unless SiteConfig.collective_noun_disabled.false?
+
+      SiteConfig.where(var: "community_name").find_each do |community_name|
+        community_name.update!(value: "#{community_name.value} #{SiteConfig.collective_noun}")
+      end
```

Comment:
Is the plan to add another data update script in a separate PR to drop the `collective_noun` and `collective_noun_disabled` fields?

Reply:
This is a nice, elegant solution and a great improvement on the previous data update script! 👏🏽 Nice job